projects/website/markdown/posts/2023-01-18-issue-bounties/post.mdx

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: Issue and feature bounties
+categories: ['Tuist', 'Xcode', 'Swift', 'Project generation']
+excerpt: Announcing new bounty program for our Tuist contributors
+author: fortmarek
+---
+
+Financing open source projects is _hard_. Therefore, we value every dollar you send our way to keep this project going after many years now. But we also want to ensure that the money is used to move the project further. This is why we're introducing **issue bounties**. But first:
+
+## Thank you!
+
+I'd like to call out all of our current public sponsors, you deserve a round of applause 👏:
+- Our silver sponsor [@GetStream](https://github.com/GetStream)
+- [@csjones](https://github.com/csjones)
+- [@GlebRadchenko](https://github.com/GlebRadchenko)
+- [@SD10](https://github.com/SD10)
+- [@mflknr](https://github.com/mflknr)
+- [@shgew](https://github.com/shgew)
+
+But we value as much all the past financial contributors and anybody who spent their free or company time on the project, helping others with questions, features, or issues 💜
+
+## Announcing bounty system
+
+Our budget is no secret and you can see our expenses [at opencollective](https://opencollective.com/tuistapp#category-BUDGET). We have recently been able to cut down our monthly recurring bills which left us wondering how we can best use the financial contributions. We value the free time contributors are putting into tuist and so we want to reward them. 
+
+We will experiment with a **bounty system** – our Tuist Core team will select issues and assign rewards to them. You will find the list [here](https://github.com/tuist/tuist/discussions). We will regularly update the list and you are free to add your suggestions there. If you have yourself an issue you'd like to see resolved, you can send a donation to our project and we will declare a bounty for you.
```

Comment:
Once this PR is approved, I will create the discussion and update this link to point to the discussion.